src/plugins/GUITestBase/src/tests/common_scenarios/create_shortcut/GTTestsCreateShortcut.cpp

```diff
@@ -22,9 +22,11 @@
 #include <QDir>
 
 #if defined(Q_OS_WIN)
+// clang-format off
```

Comment:
This is some feature which really disable clang for this lines or just a comment for us with information?

Reply:
It disables clang formatter